components/camel-observation/src/main/java/org/apache/camel/observation/MicrometerObservationSpanAdapter.java

```diff
@@ -80,11 +81,13 @@ public void setTag(String key, Boolean value) {
     @Override
     public void setLowCardinalityTag(Tag key, String value) {
         observation.lowCardinalityKeyValue(key.toString(), value);
```

Comment:
Do we want to deprecate this? If we want to migrate to the attributes version we could deprecate all of the `key.toString()` options and mark that this is scheduled for removal in e.g. 3 minor releases? WDYT?